### ISSUE TITLE
Not working on M1 Macbook Air (MacOS Monterey 12.5.1)

### ISSUE THREAD

**Issue by kmacdough** [Jac-Zac/Nibbler_MacOS#9]
I am attempting to get Nibbler running with any version of lc0 on an M1 Macbook Air running MacOS 12.5.1 Monteray.

I downloaded and installed the Nov 2022 release and  ran `xattr -cr Nibbler.dmg` before opening the DMG.

Nibbler opens, and the GUI looks fine, but I've been unable to get it to connect to an lc0 engine. The documentation as written gives me the impression it should otherwise work and comes with it's own engines:

 > When you download it you have other things like a version of stokfish and lc0.

Perhaps this is meant to read "... you *should already* have other things ..."?

Regardless, I've attempted to get it running with a separate version of lc0 installed via `brew install lc0`.

I had to resort to manually editing engines.json because the finder-esque browser is unable to see system directories like '/opt' where homebrew's Cellar is located. I wasn't sure of the expected format for `config.json` though. If there was a correct starting config.json I accidentally overwrote it to point at "/Applications/nibbler.app" when attempting to search for the engines that I believed where packaged. I have tried pointing to everything from the top-level Cellar folder, the ./bin folder and the ./bin/lc0 executable itself, and all throw errors.

Clearly I'm missing something. Is there a part of the installation documentation I missed? Am I expected to already have experience setting up and configuring chess engines? Are the installations out of date?

I've attached my current config.json and engines.json.
[config.json](https://github.com/user-attachments/files/17006698/config.json)
[engines.json](https://github.com/user-attachments/files/17006699/engines.json)

**Comment by klistwan** [comment]
I agree the docs are unclear on how to set it up if Leela was installed via Homebrew. What I ended up doing is creating a symlink to a more accessible directory and it seems be working:

```zsh
ln -s "$(brew --prefix lc0)/bin/lc0" ~/Chess/lc0
```

**Comment by b0r0mir** [comment]
It's working on my m1 iMac. Make sure to press shift-cmd-dot to show hidden files when browsing for lc0 in /opt/homebrew/bin. Also I ran xattr -c /Appplications/Nibbler.app instead of on the dmg file.
